mabwiser/linear.py

```diff
@@ -45,33 +45,62 @@ def fit(self, X, y):
 
         # Update A
         self.A = self.A + np.dot(Xt, X)
-        self.A_inv = np.linalg.inv(self.A)
+
+        try:
+            self.A_inv = np.linalg.inv(self.A)
+        except np.linalg.LinAlgError:
```

Comment:
Does it make sense to return coefficients if A is not invertible?

Reply:
Reverted to old version